app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
```

Comment:
Sorry, thought you said that it raised a runtime exception. I was playing with a couple of things earlier, and threw the assert in while refactoring (instead of writing a test).